node/src/chain_spec/crab.rs

```diff
@@ -200,6 +200,7 @@ pub fn genesis_config() -> ChainSpec {
 				darwinia_staking: DarwiniaStakingConfig {
 					now: SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis(),
 					elapsed_time: 0,
+					max_unstake_ring: 10_000_000 * UNIT,
```

Comment:
What's the max deposit currently?

```suggestion
					max_unstake_ring: 20_000_000 * UNIT,
```